internal/services/manager_test.go

```diff
@@ -0,0 +1,90 @@
+package services_test
+
+import (
+	"context"
+	"flag"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+	"github.com/ubuntu/authd/internal/services"
+	"github.com/ubuntu/authd/internal/testutils"
+	"google.golang.org/grpc"
+)
+
+func TestNewManager(t *testing.T) {
+	tests := map[string]struct {
+		cacheDir string
+
+		systemBusSocket string
+
+		wantErr bool
+	}{
+		"Successfully create the manager": {},
```

Comment:
I think this is connect to the real system bus, which will be an issue on an official builder (also, it means the test is relying on the system one).

In that case, can you create a system bus at the start of the tests, just so that it can pass?

(btw, I’m aware that there are probably other places like in cmd/daemon which is failing too since we introduce this dbus dep)